desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,129 @@
+---
+description: admin controls for desktop
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure Admin Controls
+--- 
+
+>Note
+>
+>Admin Controls is currently in [Early Access](../../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
+
+This page contains information about how administrators can configure Admin Controls to specify and lock configuration parameters to create a standardized Docker Desktop environment across the organization.
+
+Admin Controls is designed specifically for organizations who don’t give developers root access to their machines.
+
+## Prerequisite
+
+You need to [configure a registry.json to enforce sign-in](../../../docker-hub/configure-sign-in.md). For this configuration to take effect, Docker Desktop users must authenticate to your organization. 
+
+## Step one: Place the `admin-settings.json` file in the correct location
```

Comment:
Depending on what we're able to get done on time for launch, we might have an alternative way of 'placing the admin file in the correct location', which will be via an 'admin-settings' flag on the install CLI command for windows and mac.